scheduler/src/cook/mesos/api.clj

```diff
@@ -198,9 +204,13 @@
   (-> Job
     ;; make max-runtime optional.
     ;; It is *not* optional internally but don't want to force users to set it
+    (dissoc :name)
```

Comment:
We should have unit tests verifying this behavior change of optional name and priority fields.